umei-openapi.json

```diff
@@ -989,12 +989,106 @@
         }
       }
     },
+    "/PublicOrders": {
+      "get": {
+        "tags": [
+          "Order"
+        ],
+        "summary": "View publicly available order information",
+        "description": "This endpoint returns information about publicly visible flexibility orders. Different FMOs might expose different information. Each 'Order' returned in this endpoint represents either a single order, or multiple orders that have been aggregated together.  ",
```

Comment:
Aggregated orders ? Will it not rather be aggregated trades ?

Reply:
No... aggregated orders? E.g. if there are two orders, each offering 5 MW (for a given market, period etc), some FMOs, e.g. NODES, will aggregate those two orders into one 10MW order (when viewing publicly visible orders)

Reply:
Oh ok, got it. I was confused because in the PT demo it's not the responsibility of the FMO to aggregate the orders. But indeed it could be the case for other FMO. So I am ok with your description